### ISSUE TITLE
Reorg folder

### ISSUE THREAD

**Issue by tlienart** [tlienart/FranklinTemplates.jl#45]
The main goal should be that editing / adding a new template be made as easy as possible; a secondary nice goal would be that it has a natural folder structure but I care somewhat less about that as people are not really expected to read through this repo.

Functionality that should be added:

- make it easier to work on a new template and then just merge what needs to be merged.

it should be something like `newtemplate(from_path)` which would add relevant documents at the right places of `pathof(FranklinTemplates)`. If the template exists, this would overwrite or something; there should be a check that it doesn't overwrite one of the standard one; 

- make it easier to PR

basically the user should only provide a description and a screenshot that meets some requirements and that's it.

**Comment by tlienart** [comment]
the stuff to simplify one's life with the addition/change of template is in
